File: Desktop/SummerVacation/0703/python/numThirtyThree.py
import random
def switch(a) :
    return{
        0 : "Yes.",
        1 : "Maybe.",
        2 : "No.",
        3 : "Ask me later.",
    }.get(a,"")
question = input("What is the question? ")
print(switch(random.randrange(0,4)))

File: Desktop/SummerVacation/0703/src/numThirtyThree.java
import java.util.Scanner;
import java.util.Random;
public class numThirtyThree {
    public static void main(String[] args) {
        Random makeAnswer = new Random();
        Scanner scan = new Scanner(System.in);
        System.out.print("What is your question? ");
        String question = scan.next();
        int answer = makeAnswer.nextInt(4);
        switch(answer){
            case 0 : question = "Yes."; break;
            case 1 : question = "No."; break;
            case 2 : question = "Maybe."; break;
            case 3 : question = "Ask again later."; break;
        }
        System.out.print(question);
    }
}

